src/main/scala/scorex/core/newserialization/Writer.scala

```diff
@@ -0,0 +1,53 @@
+package scorex.core.newserialization
+
+trait Writer {
```

Comment:
@VictorMikheev `putOption` is missing although there is `Reader.getOption()`.

Reply:
fixed